io/disk/vfc-tests/vfc_dlpar.py

```diff
@@ -222,7 +222,7 @@ def device_add_remove(self, operation, vfc_dic):
                       vfc_dic["r_lpar"], vfc_dic["r_slot"], vfc_dic["wwpn"])
         else:
             cmd = 'chhwres -r virtualio -m %s -o r -p %s  -s %s' \
-                   % (self.server, self.lpar, vfc_dic["c_slot"])
+                % (self.server, self.lpar, vfc_dic["c_slot"])
```

Comment:
@abdhaleegit looks like some indentation here. please re-check as this a command ad it may fail on script run

Reply:
Not a problem I see when running the test